packages/fds-components/src/components/interactive/toast.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import IconButton from '@material-ui/core/IconButton';
+import CountdownButton from '../media/CountdownButton';
+
+import Toast from './Toast';
+
+const renderByType = (type, extraProps) =>
+  shallow(<Toast type={type} content={<p>lol</p>} {...extraProps} />);
+
+
+describe('Toast component', () => {
+
+  it('matches snapshot (info)', () => {
+    const component = renderByType('info');
+    expect(component).toMatchSnapshot();
+  });
+
+  it('matches snapshot (success)', () => {
+    const component = renderByType('success');
+    expect(component).toMatchSnapshot();
+  });
+
+  it('matches snapshot (progress)', () => {
+    const component = renderByType('progress');
+    expect(component).toMatchSnapshot();
+  });
+
+  it('matches snapshot (error)', () => {
+    const component = renderByType('error');
+    expect(component).toMatchSnapshot();
+  });
+
+  it('matches snapshot (warn)', () => {
+    const component = renderByType('error');
+    expect(component).toMatchSnapshot();
+  });
+
+  it('does NOT render action button if label and action are not passed', () => {
+    const component = renderByType('info');
+    const actionButton = component.find('.toast-action');
```

Comment:
might be better to use a `dataTest` prop for this

Reply:
good point, will do